### PR TITLE
Add job cancellation docs

### PR DESCRIPTION
**Pull request by pcarranza**
Simple documentation explaining how they work.

### CODE REVIEW

File: content/cancelling-jobs.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2018-02-14T13:43:27+01:00
+weight: 40
+---
+
+Once jobs are running the user has total control over them. This means that at
+any time a job can be queried for status, get the logs as they are being
+streamed, and of course, they can be cancelled.
+
+Job cancellation internally works the same way a timeout is handled. This means
+that the job will get a kill signal, and as a result it will err out, leaving
```
Comment (omame) [inline content/cancelling-jobs.md]: `s/err/error/`

File: content/cancelling-jobs.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+the final state of the job as failed.
+
+Still, any log that was streamed up to that point will still be recorded,
```
Comment (omame) [inline content/cancelling-jobs.md]: I'd remove the second still.

File: content/cancelling-jobs.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+meaning that the user can evaluate how far the command reached.
+
+## Cancelling a job
+
+* `cancel` cancels a job owned by the calling user that is currently running
```
Comment (omame) [inline content/cancelling-jobs.md]: `cancel <job id>`?

File: content/cancelling-jobs.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+* `kill` cancels a job owned by any user. This commands requires the calling user to be in the admin group.
```
Comment (omame) [inline content/cancelling-jobs.md]: `kill <job id>`?

**Review by omame — COMMENTED** [review]
Minor nitpicks.

File: content/cancelling-jobs.md (hunk 2 above)
Comment (pcarranza) [reply content/cancelling-jobs.md]: yep, makes sense.

File: content/cancelling-jobs.md (hunk 1 above)
Comment (pcarranza) [reply content/cancelling-jobs.md]: ok

**Review by omame — APPROVED** [review]

**Pull request merged**